crypto/src/main/java/org/web3j/crypto/transaction/type/Transaction1559.java

```diff
@@ -45,8 +44,22 @@ public Transaction1559(
             String data,
             BigInteger maxPriorityFeePerGas,
             BigInteger maxFeePerGas) {
-        super(EIP1559, nonce, null, gasLimit, to, value, data);
-        this.chainId = chainId;
+        super(chainId, nonce, null, gasLimit, to, value, data, Collections.emptyList());
+        this.maxPriorityFeePerGas = maxPriorityFeePerGas;
+        this.maxFeePerGas = maxFeePerGas;
+    }
```

Comment:
Even if it looks simple a test case will be worth here, also will improve the test coverage